docker/jenkins/config-sanitizer.dict

```diff
@@ -0,0 +1,23 @@
+{
+  'api_calls_timeout_seconds': 300,
+  'image': 'dalg24/dtk-stack',
+  'tag': 'latest',
+  'name': os.getenv("BUILD_TAG"),
+  'cmd': [
+    'sh -c "patch -d /scratch/source/trilinos/release -p1 -i /scratch/source/trilinos/release/DataTransferKit/docker/jenkins/trilinos_stk.patch"',
+    'sh -xe /scratch/source/trilinos/release/DataTransferKit/docker/jenkins/build-sanitizer.sh',
```

Comment:
so here same dictionary as in `docker/jenkins/config.dict` but this line that points to the alternate build script.  is that right?